source/documentation/retire-packages.html.md

```diff
@@ -0,0 +1,50 @@
+---
+author: ykarel
+title: Retiring a package
+---
+
+# Retiring a package from RDO
+
+1. toc
+{:toc}
+
+### How to remove an OpenStack package from RDO Trunk
```

Comment:
This procedure will retire it from RDO Trunk and also prevent it from being in the next CloudSIG repo. Please, explain it.

Reply:
Done